tools/pylint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This script can be used to run the pylint_runner while ignoring migrations.
+
+# Import utility functions
+# shellcheck source=./tools/_functions.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
```

Comment:
You need to fix this path to be `source "$(dirname "${BASH_SOURCE[0]}")/utils/_functions.sh"`

Reply:
Oh, yes, thank you!